assets/locales/en/main.json

```diff
@@ -59,9 +59,44 @@
     "having-trouble": "Having trouble…",
     "no-street": "No street selected.",
     "trouble-loading": "We’re having trouble loading Streetmix.",
+    "error-code": "(Error {{code}}.)",
     "slow-connection-1": "Streetmix wasn’t able to connect to the Internet in awhile now.",
     "slow-connection-2": "You might want to reload the page and try again. Please note you might lose the latest change to the street. Sorry!",
     "no-connection": "Streetmix is having trouble connecting to the Internet."
+  },
+	"errors": {
+		"buttons": {
+			"home-button": "Go to the homepage",
+			"sign-in-button": "Sign in again",
+			"reload-button": "Reload the page",
+			"try-again-button": "Try again",
+			"general-help-buttons": "<a target='_blank' href='mailto:hello@streetmix.net'>email</a> or <a target='_blank' href='https://twitter.com/intent/tweet?text=@streetmix'>Twitter</a>. <br /> <button id = 'error-home'onClick = {goHome} > Go to the homepage < /button>"
+		},
+		"page-not-found-title": "Page not found.",
+		"page-not-found-description": "Oh, boy. There is no page with this address!<br />{homeButton}",
+		"street-not-found-title": "Street not found.",
+		"street-not-found-description": "Oh, boy. There is no street with this link!<br />{homeButton}",
+		"street-not-found-but-user-description": "There is no street with this link! But you can look at other streets by {linkToUser(this.props.street)} <br />{homeButton}",
+		"street-deleted-title": "This street has been deleted.",
+		"street-deleted-description": "There is no longer a street with this link, but you can look at other streets by {linkToUser(this.props.street)} <br />{homeButton}",
+		"sign-out-title": "You are now signed out.",
+		"reload-sign-out-title": "You signed out in another window.",
+		"gallery-street-failure-description": "We’re having trouble loading this street.",
+		"reload-sign-out-description-401": "Please reload this page before continuing.<br />(Error RM2.)<br /><button id='error-clear-sign-in-reload' onClick={goReloadClearSignIn}>Reload the page</button>",
+		"reload-sign-in-title": "You signed in in another window.",
+		"reload-sign-in-description": "Please reload this page before continuing.<br />{reloadButton}",
+		"street-deleted-elsewhere-title": "This street has been deleted elsewhere.",
+		"street-deleted-elsewhere-description": "This street has been deleted in another browser.<br />{homeButton}",
+		"twitter-access-denied-title": "You are not signed in.",
+		"twitter-access-denied-description": "You cancelled the Twitter sign in process.<br />{homeButton}",
+		"auth-api-problem-title": "There was a problem with signing you in.",
+		"auth-api-problem-description": "There was a problem with Twitter authentication. Please try again later or let us know via {generalHelpButtons}",
```

Comment:
Let's break up the "Please try again later" message into a new string and instead of using `{generalHelpButtons}` let's bake that string right in to the message, like so:

```
Please try again later or let us know via <a target='_blank' href='mailto:hello@streetmix.net'>email</a> or <a target='_blank' href='https://twitter.com/intent/tweet?text=@streetmix'>Twitter</a>.
```

To make our lives easier, we should replace the URLs with placeholders.

```
Please try again later or let us know via <a target='_blank' href='{email}'>email</a> or <a target='_blank' href='{twitter}'>Twitter</a>.
```

Reply:
Do we want to keep the "Please try again later" in the `error` key or put it in the `msg` key? Other question here, should we remove the URLs or just keep them there since Transifex already detects them?

Reply:
And then should we use {} or {{}} as placeholders? On the Transifex end, should we tell it to detect both for in Transifex UI?

Reply:
I'll try to keep things simple.

1. We will _eventually_ use single braces `{}`.
2. We _currently_ use double braces `{{}}`.

Let's _currently_ be consistent and use double braces for everything.

When we are ready to switch to single braces, then we will adjust the strings. Although, I'm not sure if adjusting the strings will erase the old translations, so let's be aware that that might be the case.